### ISSUE TITLE
can't range map interface

### ISSUE THREAD

**Issue by leonxiong516** [containous/yaegi#529]
The following program `sample.go` triggers a panic:

```go
package main

func main() {
        m := make(map[string]interface{})
	m["A"] = 1
	for _, v := range m {
		fmt.Println(v)
	}
}
```
Got:
```console
$ yaegi ./sample.go
 reflect.Set: value of type interface {} is not assignable to type interp.valueInterface
```

**Comment by mvertes** [comment]
Thanks for your report.

The test case relates to range of map interface only. Could you open a separate issue with appropriate descritption and test case (if possible) regarding the multi-gopath problem that you mention in the issue title ?

Thanks!

**Comment by mvertes** [comment]
Fixed by #528
